src/components/form/helper.ts

```
@@ -0,0 +1,100 @@
+			} else if (isObjectField(prop)) {
+				value = JSON.stringify(prop.default);
+			} else if (isArrayField(prop)) {
+				value = prop.default;
+			} else {
+				value = prop.default.toString();
+			}
+		} else if (isArrayField(prop)) {
+			value = [];
+		}
+
+		return { ...acc, [key]: value };
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)